tests/e2e/transitions/public_form_transitions.spec.js

```diff
@@ -91,6 +91,34 @@ const transitionsConfig = {
       }],
     }]
   }],
+  update_clinics: [ {
+    form: 'TEMP',
```

Comment:
To make these e2e tests have better coverage, some of these form configs should be omitted.